content/engineering/enablement/dev-experience/index.md

```diff
@@ -55,3 +48,31 @@ We aim to allow teammates the flexibility to work on incoming requests, tackle p
 _This section is a work in progress._
 
 We currently have weekly sync meetings and biweekly retrospectives.
+
+### Support
+
+Support is handled through the `@dev-experience-support` handle in Slack.
+Support on-call responsibilities on this team include:
+
+- Urgent questions and issues
+- [Build pipeline support](#build-pipeline-support)
+
+#### Build pipeline support
+
+Build pipeline support pertains to our [continuous integration](https://docs.sourcegraph.com/dev/background-information/continuous_integration).
+This responsibility can be described as that of a "build sheriff" - the goal is to have someone lead on identifying the right person to drive a fix on an issue, rather than actively fixing every issue that arises.
```

Comment:
Is it confusing to refer to a "build sheriff" here? My thought is that the build sheriff is what we're proposing for a separate rotation across engineering teams, whereas this documentation will be for our specific dev-experience-support rotation.

Reply:
Oh - I was thinking it would be the same rotation 😅 Is there a reason to create a separate rotation? "build sherrif" seems to be an exact overlap of support responsibilities, which is interfacing with other teams for questions and issues

Reply:
Nevermind - I just saw [RFC 515](https://docs.google.com/document/d/1rHOOgvWmBB5c4aS_wWPogNCAWT6_tww8tceSy6nzFy8/edit#)! Since that will involve other teams, let's keep this here as part of support responsibilities in the interim nd update it when we get thumbs-up from other teams?